Src/IronPython.Modules/time.cs

```diff
@@ -71,7 +71,6 @@ public static class PythonTime {
 
             // altzone, timezone are offsets from UTC in seconds, so they always fit in the
             // -13*3600 to 13*3600 range and are safe to cast to ints
-#if FEATURE_TIMEZONE
```

Comment:
You want to get rid of the TimeZone stuff and keep the TimeZoneInfo stuff. See IronLanguages/ironpython2#203.